sklearn/multioutput.py

```diff
@@ -547,12 +547,13 @@ class ClassifierChain(MetaEstimatorMixin, ClassifierMixin, _BaseChain):
         - An iterable yielding (train, test) splits as arrays of indices.
 
     random_state : int, RandomState instance or None, optional (default=None)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
-
-        The random number generator is used to generate random chain orders.
+        If ``order=random``, controls the random seed given to generate random
+        chain orders.
+        In addition, it controls the random seed given at each `base_estimator`
+        at each chaining iteration.
+        Thus, it is only used when `base_estimator` expose a `random_state`.
```

Comment:
I'd remove the line break since the second sentence is just a additional info about the first one.